spcs-three-tier/app/manifest.yml

```diff
@@ -8,6 +8,7 @@ configuration:
 
 artifacts:
   setup_script: setup.sql
+  readme: README.md
```

Comment:
The file actually has a different case (in app/):
```suggestion
  readme: readme.md
```

Reply:
done :D